### PR TITLE
[Website] Ft Partners frontend

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)
- Features design of partner details page.
- Dynamic display of partners on about-us page.
- CTA links added to two sections: _Meet the team_ and _Our partners_

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* cd website
* npm run standalone (node v12)
* Route to _/about-us_ and click a partner image 

#### What are the relevant tickets?
- [WEB-331]

#### Screenshots
Screenshot (81)
Screenshot (80)

[WEB-331]: https://airqoteam.atlassian.net/browse/WEB-331?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @FaithDaka 
LGTM

File: website/frontend/src/pages/AboutUsPage.js [hunk 1]
```diff
@@ -21,13 +21,13 @@ import HorizonteCiudadanoLogo from 'assets/img/partners/Horizonte Ciudadano Foun
...
 import MappingForChangeLogo from 'assets/img/partners/Mapping for Change UK.png';
-import MinistriesLogo from 'assets/img/partners/Ministries.png'
+import MinistriesLogo from 'assets/img/partners/Ministries.png';
 import MUKLogo from 'assets/img/partners/MUK_Logo.png';
 import NEMALogo from 'assets/img/partners/NEMA.png';
 import SheffieldUniversityLogo from 'assets/img/partners/Sheffield University UK.png';
 import UNEPLogo from 'assets/img/partners/UNEP.png';
 import UniversityOfColoradoBoulderLogo from 'assets/img/partners/University of Colorado Boulder US.png';
-import UniversityOfFloridaLogo from 'assets/img/partners/University of Florida US.png' 
+import UniversityOfFloridaLogo from 'assets/img/partners/University of Florida US.png';
```
Comment (Baalmart) [inline website/frontend/src/pages/AboutUsPage.js:30]: Thanks @FaithDaka , I guess it is just safer not to have white spaces in file names. https://superuser.com/questions/29111/what-technical-reasons-exist-for-not-using-space-characters-in-file-names

**Review by Baalmart — COMMENTED** [review]
Thanks @FaithDaka , all good, just left a small minor comment for your review.

File: website/frontend/src/pages/AboutUsPage.js (hunk 1 above)
Comment (FaithDaka) [reply website/frontend/src/pages/AboutUsPage.js]: All the unused imports have been removed @Baalmart

**Review by Codebmk — APPROVED** [review]
LGTM. Thanks.

File: website/frontend/src/pages/AboutUsPage.js (hunk 1 above)
Comment (Baalmart) [reply website/frontend/src/pages/AboutUsPage.js]: Thanks @FaithDaka

**Comment by Baalmart** [comment]
Hi @FaithDaka , looks like some new merge conflicts have just shown up. Please address them and we move forward. Thanks again!

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**